tests/software/fgbio/callmolecularconsensusreads/main.nf

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { FGBIO_SORTBAM } from '../../../../software/fgbio/sortbam/main.nf' addParams( options: [args: '-s TemplateCoordinate', suffix: '_out'] )
+include { FGBIO_CALLMOLECULARCONSENSUSREADS } from '../../../../software/fgbio/callmolecularconsensusreads/main.nf' addParams( options: [args: '-M 1', suffix: '_molreads'] )
+
+workflow test_fgbio_callmolecularconsensusreads {
+    
+    def input = []
+    input = [ [ id:'test' ], // meta map
+              file('https://github.com/fulcrumgenomics/fgbio/blob/master/src/test/resources/com/fulcrumgenomics/bam/200reads.bam?raw=true', checkIfExists: true) ]
```

Comment:
Please use the test files provided in the modules respository. Please have a look at `tests/config/test_data.config` and use `params.test_data` to access the files.

Reply:
Thank you for the feedback! I fixed the data for both modules.